lms/envs/common.py

```diff
@@ -1903,6 +1903,12 @@
     'ROOT_PATH': '/tmp/edx-s3/grades',
 }
 
+FINANCIAL_REPORTS = {
+    'STORAGE_TYPE': 'localfs',
+    'BUCKET': 'edx-grades',
+    'ROOT_PATH': '/tmp/edx-s3/grades',
```

Comment:
For FINANCIAL_REPORTS, can we have a different BUCKET and ROOT_PATH than DOWNLOAD_GRADES?